Dockerfile

```diff
@@ -11,12 +11,20 @@ WORKDIR /dojo
 COPY . .
 RUN cargo build --release --config net.git-fetch-with-cli=true
 
+# Set an environment variable for the port
+ENV HEALTH_CHECK_PORT=5050
+
 FROM debian:buster-slim
 LABEL description="Dojo is a provable game engine and toolchain for building onchain games and autonomous worlds with Cairo" \
     authors="tarrence <tarrence@cartridge.gg>" \
     source="https://github.com/dojoengine/dojo" \
     documentation="https://book.dojoengine.org/"
 
+HEALTHCHECK --interval=3s --timeout=5s --start-period=1s --retries=5 \
```

Comment:
isn't this only a `katana` healthcheck? I am not familiar with the dojo stack, but what if I use `sozo` as an entrypoint, whould this make sense?

Reply:
That's a good point, I think if we want to have the healthcheck for `katana` only, we will need to make it in a separate Dockerfile. What do you think is the best approach here @tarrencev @kariy ?